.github/workflows/publish.yml

```diff
@@ -26,6 +26,6 @@ jobs:
         run: npm ci
 
       - name: Publish package to npmjs
-        run: npm publish
+        run: npm publish --access public
```

Comment:
I'm not sure if needed or not, but in `actions/setup-node`, there is a parameter called `scope`, should we add it as well?
```
scope: '@nimblehq'
```

An example from Github: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#example-workflow

Reply:
@hoangmirs I guess we don't need to add that as it'll add scope to `.npmrc` which would be used for scoped registry 😄